docs/src/scheduling.md

```diff
@@ -18,22 +18,18 @@ section in the definition of the cluster, which supports:
 - node selectors
 - tolerations
 
-!!! Info
-    CloudNativePG does not support pod templates for finer control
-    on the scheduling of workloads. While they were part of the initial concept,
-    the development team decided to postpone their introduction in a newer
-    version of the API (most likely v2 of CNPG).
+## Pod Affinity and Anti-Affinity
 
-## Pod affinity and anti-affinity
+Kubernetes provides mechanisms to control where pods are scheduled using
+*affinity* and *anti-affinity* rules. These rules allow you to specify whether
+a pod should be scheduled on particular nodes (*affinity*) or avoided on
+specific nodes (*anti-affinity*) based on the workloads already running there.
```

Comment:
The existing text was

> on the actual workloads already running in those nodes.

And IMO I'd keep it that way instead of the "there" ... more verbose but easier to parse.